### PR TITLE
[AppService] Fix #13261 az webapp list-runtimes use static list until new Available Stacks API is available

### LINKED ISSUE #13261: Update az webapp list-runtimes to use static list until new Available Stacks API is available
- `az webapp list-runtimes` depends on the available stacks API
  - Available stacks API is updated with Antares deployments
  - Antares deployments are infrequent and don't line up with stacks EOL schedule.
- There are bugs in existing **Available Stacks API** around **LTS** and **Current** decorations
  - See `"DOTNETCORE|LTS"` and `"DOTNETCORE|Latest"`

### Hide-Windows

- "node|0.6",
- "node|0.8",
- "node|0.10",
- "node|0.12",
- "node|4.8",
- "node|6.5",
- "node|6.9",
- "node|6.12",
- "node|7.10",
- "node|8.1",
- "node|8.4",
- "node|8.5",
- "node|8.9",
- "node|8.10",
- "node|8.11",
- "php|5.6",
- "python|2.7",

### Add-Windows

- "DOTNETCORE|2.1",
- "DOTNETCORE|3.1",

### Expected results

``` bash

az webapp list-runtimes
[
  "aspnet|V4.7",
  "aspnet|V3.5",
  "DOTNETCORE|2.1",
  "DOTNETCORE|3.1",
  "node|10.0",
  "node|10.6",
  "node|10.14",
  "node|10.15",
  "php|7.2",
  "php|7.3",
  "php|7.4",
  "python|3.6",
  "java|1.7|Tomcat|7.0",
  "java|1.7|Tomcat|8.0",
  "java|1.7|Tomcat|8.5",
  "java|1.7|Tomcat|9.0",
  "java|1.7|Jetty|9.1",
  "java|1.7|Jetty|9.3",
  "java|1.7|Java SE|8",
  "java|1.8|Tomcat|7.0",
  "java|1.8|Tomcat|8.0",
  "java|1.8|Tomcat|8.5",
  "java|1.8|Tomcat|9.0",
  "java|1.8|Jetty|9.1",
  "java|1.8|Jetty|9.3",
  "java|1.8|Java SE|8",
  "java|11|Tomcat|7.0",
  "java|11|Tomcat|8.0",
  "java|11|Tomcat|8.5",
  "java|11|Tomcat|9.0",
  "java|11|Jetty|9.1",
  "java|11|Jetty|9.3",
  "java|11|Java SE|8"
]
```

### Hide-Linux

- "RUBY|2.4.5",
- "RUBY|2.3.8",
- "NODE|lts",
- "NODE|8-lts",
- "NODE|6-lts",
- "NODE|4.4",
- "NODE|4.5",
- "NODE|4.8",
- "NODE|6.2",
- "NODE|6.6",
- "NODE|6.9",
- "NODE|6.10",
- "NODE|6.11",
- "NODE|8.0",
- "NODE|8.1",
- "NODE|8.2",
- "NODE|8.8",
- "NODE|8.9",
- "NODE|8.11",
- "NODE|8.12",
- "NODE|9.4",
- "PHP|5.6",
- "DOTNETCORE|1.0",
- "DOTNETCORE|1.1",
- "DOTNETCORE|2.0",
- "DOTNETCORE|2.2",
- "DOTNETCORE|3.0",
- "DOTNETCORE|LTS",
- "DOTNETCORE|Latest",
- "WILDFLY|14-jre8",
- "PYTHON|2.7"

### Add-Linux

- "DOTNETCORE|3.1",

### Expected Results Linux

``` bash
az webapp list-runtimes --linux

[

  "DOTNETCORE|2.1",
  "DOTNETCORE|3.1",
  "NODE|12-lts",
  "NODE|10-lts",
  "NODE|10.1",
  "NODE|10.10",
  "NODE|10.12",
  "NODE|10.14",
  "NODE|10.16",
  "NODE|12.9",
  "JAVA|8-jre8",
  "JAVA|11-java11",
  "TOMCAT|8.5-jre8",
  "TOMCAT|9.0-jre8",
  "TOMCAT|8.5-java11",
  "TOMCAT|9.0-java11",
  "PHP|7.2",
  "PHP|7.3",
  "PYTHON|3.8",
  "PYTHON|3.7",
  "PYTHON|3.6",
  "RUBY|2.5.5",
  "RUBY|2.6.2",
]
```

### PR DESCRIPTION
**Pull request by calvinsID**
**Description**

Currently `az webapp list-runtimes` returns a list of runtimes provided by the Available Stacks API. However, the Available Stacks API is updated with Antares deployments which are infrequent and don't line up with stacks EOL schedule.

We want to return a list of static runtimes until the Available Stacks API is updated.

**Testing Guide**  

`az webapp list-runtimes` (--linux) should return the list shown in https://github.com/Azure/azure-cli/issues/13261

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
AppService

**Comment by panchagnula** [comment]
@btardif FYI

File: src/azure-cli/azure/cli/command_modules/appservice/_params.py [hunk 1]
```diff
@@ -252,7 +252,7 @@ def load_arguments(self, _):
             c.argument('php_version', help='The version used to run your web app if using PHP, e.g., 5.5, 5.6, 7.0')
             c.argument('python_version', help='The version used to run your web app if using Python, e.g., 2.7, 3.4')
             c.argument('net_framework_version', help="The version used to run your web app if using .NET Framework, e.g., 'v4.0' for .NET 4.6 and 'v3.0' for .NET 3.5")
-            c.argument('linux_fx_version', help="The runtime stack used for your linux-based webapp, e.g., \"RUBY|2.3\", \"NODE|6.6\", \"PHP|5.6\", \"DOTNETCORE|1.1.0\". See https://aka.ms/linux-stacks for more info.")
+            c.argument('linux_fx_version', help="The runtime stack used for your linux-based webapp, e.g., \"RUBY|2.5.5\", \"NODE|10.14\", \"PHP|7.2\", \"DOTNETCORE|2.1\". See https://aka.ms/linux-stacks for more info.")
```
Comment (panchagnula) [inline src/azure-cli/azure/cli/command_modules/appservice/_params.py:255]: @btardif can you confirm if we still need link to the aka.ms link we have in the help here for Linux runtimes

File: src/azure-cli/azure/cli/command_modules/appservice/commands.py [hunk 2]
```diff
@@ -89,7 +89,7 @@ def load_command_table(self, _):
         g.custom_command('start', 'start_webapp', validator=validate_app_or_slot_exists_in_rg)
         g.custom_command('restart', 'restart_webapp', validator=validate_app_or_slot_exists_in_rg)
         g.custom_command('browse', 'view_in_browser')
-        g.custom_command('list-runtimes', 'list_runtimes')
+        g.custom_command('list-runtimes', 'list_runtimes_hardcoded')
```
Comment (panchagnula) [inline src/azure-cli/azure/cli/command_modules/appservice/commands.py:92]: @calvinsID (Nit) can you add a comment here stating that we can move back to the original method once the backend API is updated & deployed

Comment (calvinsID) [reply src/azure-cli/azure/cli/command_modules/appservice/commands.py:93]: Added!

**Comment by panchagnula** [comment]
@calvinsID just make sure az webapp up tests pass as well since they need to be run live to verify.

**Comment by calvinsID** [comment]
@panchagnula Just added the yaml recordings from az webapp up tests

**Comment by Juliehzl** [comment]
> @panchagnula Sisira Panchagnula FTE Just added the yaml recordings from az webapp up tests

Hi @calvinsID , where is your test python file besides yaml?

**Comment by panchagnula** [comment]
> > @panchagnula Sisira Panchagnula FTE Just added the yaml recordings from az webapp up tests
> 
> Hi @calvinsID , where is your test python file besides yaml?

@Juliehzl these are live tests do we need to include the yaml files for these & what is the test python file you are referring to? Thanks!

**Comment by calvinsID** [comment]
> > > @panchagnula Sisira Panchagnula FTE Just added the yaml recordings from az webapp up tests
> > 
> > 
> > Hi @calvinsID , where is your test python file besides yaml?
> 
> @Juliehzl these are live tests do we need to include the yaml files for these & what is the test python file you are referring to? Thanks!

I removed the yaml recordings since they were generated from running live tests - there were no newly created/modified tests.

File: src/azure-cli/azure/cli/command_modules/appservice/_params.py (hunk 1 above)
Comment (btardif) [reply src/azure-cli/azure/cli/command_modules/appservice/_params.py:255]: Keep this link for now, I'm going to try and update it to point somewhere else.

Comment (btardif) [reply src/azure-cli/azure/cli/command_modules/appservice/_params.py:255]: @panchagnula  I was able to take ownership of the link and it's now pointing to: https://github.com/Azure/app-service-linux-docs/tree/master/Runtime_Support 

so yes, lets keep this.

**Comment by panchagnula** [comment]
@Juliehzl could you review please? Thanks!

**Comment by Juliehzl** [comment]
Can we add some checks here?

https://github.com/Azure/azure-cli/blob/fe18bc1c9832da8f0d1dc9082d14bbcd800926e5/src/azure-cli/azure/cli/command_modules/appservice/tests/latest/test_webapp_commands.py#L126-L127

**Comment by yonzhan** [comment]
add to S171

**Pull request merged**